### DISCUSSION TITLE
Add `--serde-serialize-rename-all` flag to `sea-orm-cli generate entity`

### DISCUSSION THREAD

**Discussion by ferreira-tb** [SeaQL/sea-orm#2144] [Ideas]
First of all, thanks for this awesome crate. I'm currently experimenting with using it in my Tauri apps and, so far, it's been a very pleasant experience.

I'm generating my entities using the `--with-serde` flag, as Tauri requires types to implement [serde::Serialize](https://docs.serde.rs/serde/trait.Serialize.html) so we can return them from commands. However, I'd like to serialize them as `camelCase`, as that is the convention in TypeScript.

This can easily be done manually:

```rust
#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq, Serialize, Deserialize)]
#[sea_orm(table_name = "character")]
#[serde(rename_all(serialize = "camelCase"))] // Adding this
pub struct Model {
  #[sea_orm(primary_key)]
  pub id: i32,
}
```

Would be really nice if a new flag is added so that line is added automatically.

**Comment by anshap1719** [comment]
@ferreira-tb You can do it by passing `--model-extra-attributes="#[serde(rename_all = \"camelCase\")]"`

**Reply by ferreira-tb** [reply]
Oh, thanks! I missed that
